cg/meta/workflow/utils/utils.py

```diff
@@ -9,3 +10,14 @@ def are_all_samples_control(case: Case) -> bool:
         sample.control in [ControlOptions.NEGATIVE, ControlOptions.POSITIVE]
         for sample in case.samples
     )
+
+
+def map_to_trailblazer_priority(priority: int) -> TrailblazerPriority:
+    """Map a priority to a Trailblazer priority."""
+    return {
+        0: TrailblazerPriority.LOW,
+        1: TrailblazerPriority.NORMAL,
+        2: TrailblazerPriority.HIGH,
+        3: TrailblazerPriority.EXPRESS,
+        4: TrailblazerPriority.NORMAL,
+    }[priority]
```

Comment:
```suggestion
MAP_TO_TRAILBLAZER_PRIORITY: dict[int, TrailblazerPriority] = {
        0: TrailblazerPriority.LOW,
        1: TrailblazerPriority.NORMAL,
        2: TrailblazerPriority.HIGH,
        3: TrailblazerPriority.EXPRESS,
        4: TrailblazerPriority.NORMAL,
    }
```
Probably just flavour so feel free to keep it as is. The only difference will be whether you get the priority as `map_to_trailblazer_priority(case.priority)` or `MAP_TO_TRAILBLAZER_PRIORITY[case.priority]`

Reply:
Also - I have no idea how the clinical_trials option 4 should be mapped, but given that it maps to "normal" slurm qos, this probably makes sense